CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [7.5.1] - 2024-04-??
 
+- Added: The Discord bot now mentions the game when describing a preset.
```

Comment:
I thought about if this is relevant because it doesn't touch the applicaiton.
Looking into the logs, it seems to be mentioned but we had a `Discord Bot` section often but sometimes not.
Dunno if we want to reintroduce that.

Reply:
I thought might be better just in case:tm:. Since it would affect people hosting the bot

Reply:
I already mentioned that it was done in before but what about the second part?

Reply:
I'd say a section "Discord Bot" similar to the games. IMO these are more so our users will know differences in the bots.